MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj

```diff
@@ -80,6 +80,11 @@
     <PackageReference Version="4.0.1" Include="SharpDX.XInput" />
   </ItemGroup>
 
+  <!-- NetCore3.1, NetStandard2.1 and Net5 onwards already include Vectors-->
```

Comment:
I feel its self explanatory why a reference to System.Numerics exist, don't think we want comments like these in the .csproj. @Jjagg Thoughts?

Reply:
Yeah, these should be removed.

Reply:
Referencing System.Numerics.Vectors has its tricks.... it only needs to be referenced below certain framework versions. Because it's in by default on higher versions. Also, Xamarin provides its own assembly, that's why in xamarin platforms the package should not be referenced in favour of the assembly reference. I don't think everybody is aware of these details, so if another contributor comes in and changes the frameworks versions without realizing that they should change the vectors reference...

Reply:
Build bot will stop them, don't worry :)

Not having to reference stuff when you get to new enough framework is quite normal for Microsofts system packages.